test/templates/common.py

```diff
@@ -0,0 +1,62 @@
+import subprocess
+import sys
+import re
+import os
+
+_me = os.path.basename(__file__)
+v = 0
```

Comment:
I think we should avoid acronyms if possible. Maybe use `verbose` or `debug` here? And also a boolean instead of number? :)

Reply:
+1 for full name. But verbosity may have different levels (usually depending on number of times the user passed `-v` on command line, we do not yet have this but maybe in future) so I'd suggest to keep it int.

Reply:
I thought maybe that was your intent. Good point +1. Then add full name and keep it as a integer.